yarn.lock

```diff
@@ -4441,6 +4441,14 @@
   resolved "https://registry.yarnpkg.com/@zeit/next-source-maps/-/next-source-maps-0.0.4-canary.1.tgz#5051ff8425e5f615da2d21dd08a99284fbb63d7d"
   integrity sha512-SPQCLs7ToaqzQnqXqGSCoL7KTlnOAao+1F5hy7Hkuq85TjHsUC3eeLsmVrBIraIhXG/ARHmZ0JHOesPDtBfpzw==
 
+"@zooniverse/react-components@~0.7.1":
```

Comment:
I think this is being added to the lock file because it's a dev dependency of the classifier.